pkg/satellite/stream/stream.go

```diff
@@ -181,7 +182,36 @@ func (ss *satelliteStream) ackReceivedTelemetry(telemetryMessageAckId string) {
 	}
 }
 
+func (ss *satelliteStream) performAutoClose() {
+	log.Printf("Stream auto-close conditions met - exiting")
+	if ss.showStats {
+		metrics.logReport()
+	}
+	close(ss.closeTickerChan)
+	ss.Close()
+	os.Exit(0)
+}
+
 func (ss *satelliteStream) recvLoop() {
```

Comment:
Abbreviations are only allowed for easy to understand and well established shortenings of words. `recv` is not that. I can't tell if it's `receive` or `recover` or something else.

Remember the scouts rule. If it can be refactored and cleaned up easily while working on existing code, definitely do it.

Reply:
Btw. If a method name has this issue, then it's very likely there will be other variables with the same issue nearby.

Reply:
Attempting to add context without taking a side: the spelling `recv` is widely used in the [gRPC package](https://pkg.go.dev/google.golang.org/grpc#ClientStream). Somebody was probably fixated on having the same number of characters in the names `send` and `recv`.

Reply:
Yeah that's not a good enough justification, for the resulting poorer readability.

Reply:
It is difficult for me to keep my eye open for this kind of changes, as I am often focused only on the changes that I'm making. Is this really necessary?

Reply:
> It is difficult for me to keep my eye open for this kind of changes, as I am often focused only on the changes that I'm making.

That's why code reviews are a great way to get an extra pair of eyes helping.

> Is this really necessary?

Yes. Refactoring is something which every developer has to be comfortable with at all times.
Assuming the environment allows for refactoring.
As we discussed in the morning today. We can't do much refactoring for the logic, due to the lack of proper tests written. However for un-exported methods like this one, and locally scoped variables it is safe to fix up names and continue perpetuating good culture in the team.

Reply:
I will do a pass on the stream.go file looking for incorrect / too abbreviated naming. I'll use my best judgement on what to change.